x/campaign/keeper/invariants.go

```diff
@@ -113,14 +113,6 @@ func CampaignSharesInvariant(k Keeper) sdk.Invariant {
 			campaignID := campaign.CampaignID
 			expectedAllocatedSharesShares := accountSharesByCampaign[campaignID]
 
-			campaign, found := k.GetCampaign(ctx, campaignID)
```

Comment:
*note*: this was redundant, we were already getting all campaigns from the store and iterating over all campaigns in the loop. No need to access the KVStore again (and check existence)